Makefile

```diff
@@ -357,3 +366,12 @@ hack-expose-apiserver:
 .PHONY: hack-unexpose-apiserver
 hack-unexpose-apiserver:
 	@kill -TERM $$(cat /tmp/brigade-apiserver.PID)
+
+# Convenience targets for loading images into a KinD cluster
+.PHONY: hack-load-images
+hack-load-images: load-apiserver load-scheduler load-observer load-logger-linux load-git-initializer load-worker
+
+load-%:
+	@echo "Loading $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG)"
+	@kind load docker-image $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG) \
+			|| echo >&2 "kind not installed or error loading image: $(DOCKER_IMAGE_PREFIX)$*:$(IMMUTABLE_DOCKER_TAG)"
```

Comment:
Whoa! I didn't know about `kind load`. That is really neat and useful and now I am contemplating ways I can use this to streamline my own local workflow!